cmd/thanos/receive.go

```diff
@@ -158,6 +158,7 @@ func runReceive(
 		dialOpts = append(dialOpts, grpc.WithDefaultCallOptions(grpc.UseCompressor(conf.compression)))
 	}
 	if receiveMode == receive.RouterOnly {
+		dialOpts = append(dialOpts, grpc.WithIdleTimeout(time.Duration(*conf.maxBackoff)))
```

Comment:
Is there a cmd flag to change `conf.maxBackoff`? If no, better to add one so that we can tune it later without rebuilding an image.